src/index.js

```diff
@@ -1,63 +1,33 @@
 import './style.css';
-
-const tasksList = document.getElementById('tasks-list');
-
-class Task {
-  constructor(description, completed, id) {
-    this.description = description;
-    this.completed = completed;
-    this.id = id;
-  }
-
-  static tasks = [];
-
-  updateList() {
-    this.foo = 'eslint use this';
-    tasksList.innerHTML = '';
-    const orderedTasks = Task.tasks.sort((a, b) => (a.id > b.id ? 1 : -1));
-    orderedTasks.forEach((task) => {
-      if (task.completed === true) {
-        tasksList.insertAdjacentHTML('beforeend', `
-        <div class='task-item'>
-        <p><i class="fa-regular fa-square-check"></i></p>
-        <input value='${task.description}'>
-        <p><i class="fa-solid fa-ellipsis-vertical"></i></p>
-        </div>
-        `);
-      } else {
-        tasksList.insertAdjacentHTML('beforeend', `
-        <div class='task-item'>
-        <p><i class="fa-regular fa-square"></i></p>
-        <input value='${task.description}'>
-        <p><i class="fa-solid fa-ellipsis-vertical"></i></p>
-        </div>
-        `);
-      }
-    });
-  }
-
-  addTask() {
-    Task.tasks.push(this);
-    this.updateList();
-  }
-}
+import Task from './task.js';
 
 new Task().updateList();
 
 // FOR TESTING DYNAMIC UPDATE OF THE LIST
 // ADD FIVE TASKS TO THE LIST
-// TRY CHANGING THE BOOLEAN PARAMETER! SEE THAT THE CHECK ICON UPDATES!
-// LOOK AT THE ORDER OF "id" PROPERTY. IT IS NOT ORDERED. ON "updateList()" I ORDER THE ARRAY
-// BEFORE SHOWING THE CONTENT
+// REMOVE 2 TASKS BY GIVING THE ID
+// EDIT THE DESCRIPTION OF A TASK BY GIVING THE ID AND THE MODIFIED STRING
+// SET THE STATUS OF A TASK BY GIVING THE ID AND THE BOOLEAN VALUE
 
 window.addEventListener('load', () => {
-  new Task('Task number 4', false, 4).addTask();
+  new Task('Task number 0').addTask();
+
+  new Task('Task number 1').addTask();
+
+  new Task('Task number 2').addTask();
+
+  new Task('Task number 3').addTask();
+
+  new Task('Task number 4').addTask();
+
+  new Task().removeTask(0); // Remove 'Task number 0'
 
-  new Task('Task number 2', true, 2).addTask();
+  new Task().removeTask(1); // Remove 'Task number 2' because 'Task number 1' is now id=0
 
-  new Task('Task number 0', false, 0).addTask();
+  new Task().editDescription(1, 'Task ID=1 edited');
 
-  new Task('Task number 1', true, 1).addTask();
+  new Task().setStatus(2, true);
 
-  new Task('Task number 3', true, 3).addTask();
+  // If you refresh the page, you will see that localStorage works
+  // and the array will be bigger
```

Comment:
- [ ] Great job splitting the code into multiple files. :+1: Kindly remove these lines because they are creating the tasks automatically. The application is supposed to create a task by accepting input from the text field. Please wire up the functionalities you created with the UI so a user can CREATE, EDIT, and DELETE tasks from the user interface. 🤝️

Reply:
Hello @amejid!

Thank you for your suggestions! I didn't realize that!

I have worked hard on getting your suggestion done!